src/Nri/Colors.elm

```diff
@@ -66,6 +102,56 @@ import Css exposing (hex, rgba)
 
 {-|
 
+<p style="font-size:2em; color: #00cbeb">#00cbeb</p>
+
```

Comment:
Mild opinion: our internal styleguide

Little weird to publish an elm package that anyone can use that is coupled to our specific usage of it. But then, maybe it's weird to publish colors as an elm package!